app/controllers/passwords_controller.rb

```diff
@@ -0,0 +1,6 @@
+class PasswordsController < Devise::PasswordsController
+  protected
+  def after_sending_reset_password_instructions_path_for(resource)
```

Comment:
Unused method argument - `resource`. If it's necessary, use `_` or `_resource` as an argument name to indicate that it won't be used. You can also write as `after_sending_reset_password_instructions_path_for(*)` if you want the method to accept any arguments but don't care about them.